docs/querying/sql.md

```diff
@@ -607,7 +641,7 @@ Properties connectionProperties = new Properties();
 try (Connection connection = DriverManager.getConnection(url, connectionProperties)) {
   try (
       final Statement statement = connection.createStatement();
-      final ResultSet resultSet = statement.executeQuery(query)
+      final ResultSet resultSet = statement.executeQuery(query);
```

Comment:
I think the original example is better as the semicolon is not needed

Reply:
removed